feat: adding enrichments (#186)
Basis-Theory/developers.basistheory.com: docs/expressions/detokenization.mdx

```json showLineNumbers
{

>   "ssn": "{{ token: 1d08babf-456a-4bef-993d-aece3c1a2f66 | json: '$.data}}"

Reviewer: ```suggestion
  "ssn": "{{ token: 1d08babf-456a-4bef-993d-aece3c1a2f66 | json: '$.data' }}"
```

}
```

## Using Token Properties

Card, Bank, or generic tokens containing complex JSON objects can all be detokenized as well.
Detokenization of complex token data is performed by embedding the token's JSON data within the request.

> For example, say you have the `card` token:

Reviewer: This lead-in doesn't really match the example anymore - it makes me think the example will be about embedded complex json objects into the request, but it shows examples of using metadata and enrichments

> Card, Bank, or generic tokens containing complex JSON objects can all be detokenized as well.
> Detokenization of complex token data is performed by embedding the token's JSON data within the request.
> For example, say you have the `card` token:

Reviewer: This lead-in doesn't really match the example anymore - it makes me think the example will be about embedded complex json objects into the request, but it shows examples of using metadata and enrichments. Possible suggestion:

```suggestion
In addition to the token's data, all properties of a token can be used within Detokenization Expressions.
For example, say you have the `card` token:
```

#### Original Request

> <Tabs className="bt-tabs" groupId="detokenization-examples">

Reviewer: The formatting somewhere in here got messed up, I think there is an extra ` or indentation is off?
<img width="587" alt="image" src="https://github.com/Basis-Theory/developers.basistheory.com/assets/1900725/91fa75e9-a321-4e6d-8a07-4249a9022015">

    ```shell showLineNumbers
    curl "https://api.basistheory.com/reactors/d08bc998-9301-495c-a2e5-04f8dc0916b4/react" \
    -H "BT-API-KEY: key_NS21v84n7epsSc5WzoFjM6" \
    -H "Content-Type: application/json" \
    -X "POST" \
    -d '{
    "args": {
    "card": "{{ token: 815029c2-29ec-4fc2-8cd4-99feb3ee582c | json: '$.data'}}",
    "card_owner_full_name": "John Doe"

}
}'
````

  </TabItem>
</Tabs>

#### Detokenized Request

<Tabs className="bt-tabs" groupId="detokenization-examples">
  <TabItem value="proxy" label="Proxy">

    ```json showLineNumbers
    {
      "payment_method": {
      "credit_card": {
      "number": "4242424242424242",
      "month": "11",
      "year": "2025",
      "verification_value": "123",
      "full_name": "John Doe"
    },
      "retained": true
    }
    }
    ```

  </TabItem>
  <TabItem value="reactor" label="Reactor">

    ```json showLineNumbers
    {
      "args": {
      "card": {
      "number": "4242424242424242",
      "expiration_month": 11,
      "expiration_year": 2025,
      "cvc": "123"
    },
      "card_owner_full_name": "John Doe"
    },
      "configuration": {...}
    }
    ```

  </TabItem>
</Tabs>

> asdflasjfk;asdjfasd

Reviewer: ```suggestion
```
